docs/excel/performance.md

```diff
@@ -155,11 +155,18 @@ Excel.run(async (ctx) => {
 ```
 
 > [!NOTE]
-> You can conveniently convert a Table object to a Range object by using the [Table.convertToRange()](https://dev.office.com/reference/add-ins/excel/table#converttorange) method.
+> You can conveniently convert a Table object to a Range object by using the [Table.convertToRange()](https://docs.microsoft.com/javascript/api/excel/excel.table#converttorange--) method.
+
+## Enable and disable events
+
+> [!NOTE]
```

Comment:
You probably don't need this note here -- since it appears in the Events article where the feature is actually described.